README.md

```
@@ -1,2 +1,198 @@
-# JNU-LOCKER-BE
-전남대학교 사물함 신청 서비스 플랫폼
+## JNU-LOCKER, 전남대학교 사물함 신청 서비스
+
+![](https://velog.velcdn.com/images/dubu1001/post/417a1d62-273a-4297-970e-61510f382bec/image.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)